generators/client/files-angular.js

```diff
@@ -101,7 +101,8 @@ const files = {
                 { file: '_favicon.ico', method: 'copy' },
                 '_robots.txt',
                 '_404.html',
-                '_index.html'
+                '_index.html',
+                '_manifest.webapp'
```

Comment:
our webpack build creates a `manifest.json` file. I don't want to contradict this with that. 
Also `angular mobilekit` preached  ( 😉 ) to use this name.